.bintray-gen.json

```diff
@@ -25,4 +25,4 @@
             }
```

Comment:
This file change is not necessary, and the file mode change from 644 to 755 is also necessary.